releng/org.eclipse.cdt.lsp.target/org.eclipse.cdt.lsp.target.target

```diff
@@ -34,16 +42,13 @@
 			<unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="0.0.0" />
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tm4e/releases/0.4.5/"/>
+			<repository location="https://download.eclipse.org/tm4e/snapshots/"/>
 			<unit id="org.eclipse.tm4e.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.tm4e.language_pack.feature.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/cdt/releases/10.7/" />
+			<repository location="https://ci.eclipse.org/cdt/job/cdt/job/main/lastSuccessfulBuild/artifact/releng/org.eclipse.cdt.repo/target/repository/" />
```

Comment:
@jonahgraham is this the only way to obtain CDT snapshot repository?

Reply:
Yes.